src/index.ts

```diff
@@ -38,13 +38,16 @@ interface DeclarativePDFOpts {
   defaults?: PaperOpts;
   /** Debug options (attaches parts, logs timings) */
   debug?: DebugOptions;
+  /** Title of the document */
+  documentTitle?: string;
```

Comment:
```suggestion
  /** Override for pdf document metadata and rules */
  document?: DocumentOptions;
```
Stavi ovako, pa onda dodaj DocumentOptions interface odmah iznad ovog interfacea... tu cemo predavati various document-specific stvari...

document options interface mozes odmah napraviti da je ovakav:
```ts
export interface DocumentOptions {
  /** If exists, will be used to set available metadata fields on the pdf document */
  meta?: {
    title?: string;
    author?: string;
    subject?: string;
    keywords?: string[];
    producer?: string;
    creator?: string;
    creationDate?: Date;
    modificationDate?: Date;
  };
  /**
   * Controls the minimum space the body section must occupy on each page.
   * Value is a decimal factor of the total page height (0.0 to 1.0).
   * - Default: 1/3 (a third of the page)
   * - Example: 0.25 means body must be at least 25% of page height
   */
  bodyHeightMinimumFactor?: number;
}
```

pa da nemam na svom PRu konflikte.